sirepo/template/elegant.py

```diff
@@ -742,7 +742,14 @@ def _sdds(filename):
                 f"invalid suffix={options.suffix} for download path={path}"
             )
         out = elegant_common.subprocess_output(
-            ["sddsprintout", "-columns", "-spreadsheet=csv", str(path)],
+            [
+                "sddsprintout",
+                "-noTitle",
+                "-columns",
+                "-spreadsheet=delimiter=\\,",
+                "-formatDefaults=float=%1.9e,double=%1.16e,long=%1ld,short=%1hd",
```

Comment:
I think `float=%1.8e` is better. There are at most 9 significant digits in single precision. I didn't check the short and long. 

I'm a bit concerned with the delimiter. Is there no possibility for quotes or commas in titles?

Reply:
You can't put a quote or comma in an element name, and I think that is all that would show up in the sdds output columns. I tested on several of the common elegant files with OSX Numbers and google docs. The default for spreadsheet=csv is to quote every value, which causes problems for users.

Reply:
I tested it and inspected the data. Looked fine to me.